smart-wallets/src/token/AssetToken.sol

```diff
@@ -27,8 +27,6 @@ contract AssetToken is WalletUtils, YieldDistributionToken, IAssetToken {
     struct AssetTokenStorage {
         /// @dev Total value of all circulating AssetTokens
         uint256 totalValue;
-        /// @dev Whitelist of users that are allowed to hold AssetTokens
-        address[] whitelist;
```

Comment:
Sorry we should keep the whitelist so we can call `getWhitelist`

Reply:
we can handle this offchain.